Hestia/app/src/main/java/com/rugged/application/hestia/Client.java

```diff
@@ -0,0 +1,74 @@
+package com.rugged.application.hestia; /**
```

Comment:
The javadoc will be done a lot better for the next release. But for this release, creating the javadoc is not really relevant anymore, as all code has changed. Is this ok?